ui/src/page/Login/index.js

```diff
@@ -31,12 +31,9 @@ const Login = () => {
       const credentials = btoa(username + ":" + password);
       localStorage.setItem("auth", credentials);
       localStorage.setItem("username", username);
-      localStorage.setItem("password", password);
       try {
-        const res = await getLogStream();
-        if (res.statusText === "OK") {
-          navigate("/index.html");
-        }
+        await getLogStream();
+        navigate("/index.html");
```

Comment:
But this would mean no check at all right? If the `getLogStream()` function failed wouldn't this still navigate to `index.html`

Reply:
If the request fails, it would be caught by the try catch block

Reply:
so anything other than 200 https status would be a failure here?

Reply:
4XX would be considered failure

Reply:
okay, then we're good.